### ISSUE TITLE
tutorial?

### ISSUE THREAD

**Issue by Gogothegirl999** [romen-h/kanim-explorer#5]
Hi, I would love to use this but don't know how.
I downloaded all of the stuff needed to use this but don't see any instructions or tutorial on how to use it, only features, so I would like it if that was there because I would love to see the assets animated from the game and preview them but am not sure what to use or do first...

Would love to see that added later in the future.

**Comment by romen-h** [comment]
Are you asking for a tutorial on how to use the program? Or a tutorial on how to build the program?

**Comment by romen-h** [comment]
I started a wiki to explain how to use the software.  
I can't help you with learning how to use Visual Studio if that was the original problem.

**Comment by Gogothegirl999** [comment]
> I started a wiki to explain how to use the software. I can't help you with learning how to use Visual Studio if that was the original problem.

yeah, on how to use the program because I want to know how to use it, it mentioned you can view animations of the sprite files which I want to try doing cuz I want to see certain stuff animated from the game with it's texture 2d files but nobody helped me. The only stuff I saw was to have Microsoft Visual with no instructions on why and it mentioned all the stuff you can do with the program but I saw no download .exe or any instructions on how to make it appear so...
I don't know if this program will even work for what I intend to use it for, because I want certain stuff to be animated, see that and maybe export it as gif and not use it for modding animations or whatnot, so I'm not sure if you're program does that or not....
